bin/oref0-setup.sh

```diff
@@ -278,10 +318,28 @@ if openaps vendor add --path . mmeowlink.vendors.mmeowlink 2>&1 | grep "No modul
 fi
 
 cd $directory || die "Can't cd $directory"
-if [[ "$max_iob" -eq 0 ]]; then
+if [[ "$max_iob" -eq 0 && -z "$max_daily_safety_multiplier" && -z "&current_basal_safety_multiplier" && -z "$bolussnooze_dia_divisor" && -z "$min_5m_carbimpact" ]]; then
```

Comment:
It falls through to the else clause. The specified options are used, the rest come from the defaults.